ru.linux_gaming.PortProton.yml

```diff
@@ -0,0 +1,323 @@
+app-id: ru.linux_gaming.PortProton
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+base: org.winehq.Wine
+base-version: stable-23.08
+command: portproton
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --allow=per-app-dev-shm
+  - --device=all
+  - --share=network
+  - --allow=multiarch
+  - --allow=bluetooth
+  # For Wine crash handling
+  - --allow=devel
+  - --filesystem=~/.local/share/applications:create
+  # Wine uses UDisks2 to enumerate disk drives
+  - --system-talk-name=org.freedesktop.UDisks2
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --filesystem=xdg-run/gamescope-0:ro
+  - --filesystem=host
+  # ENV
+  - --env=PATH=/app/bin:/usr/bin:/app/utils/bin:/usr/lib/extensions/vulkan/MangoHud/bin/:/usr/lib/extensions/vulkan/gamescope/bin:/usr/lib/extensions/vulkan/OBSVkCapture/bin
+  - --env=LD_LIBRARY_PATH=/app/lib:/app/lib32:/usr/lib/extensions/vulkan/gamescope/lib
+  - --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
```

Comment:
The second and the third doesn't seem right?

Reply:
Since the application uses steam runtime I copied the variables from steam to flatpak all of which are needed for correct operation

Reply:
Can you try dropping them to check if it works? I talked to the steam maintainer who added it and they said it could've been a mistake, this was added 4 years ago

Reply:
> Can you try dropping them to check if it works? I talked to the steam maintainer who added it and they said it could've been a mistake, this was added 4 years ago

I can cut to that

```
  # For Steam Deck HDR support
  - --env=LD_LIBRARY_PATH=/usr/lib/extensions/vulkan/gamescope/lib
  --env=PATH=/app/bin:/usr/bin:/app/utils/bin:/usr/lib/extensions/vulkan/MangoHud/bin/:/usr/lib/extensions/vulkan/gamescope/bin:/usr/lib/extensions/vulkan/OBSVkCapture/bin/
  ```